options/options.go

```diff
@@ -539,8 +539,9 @@ func (opts *ToolOptions) NormalizeOptionsAndURI() error {
 		return err
 	}
 
-	// connect directly, unless a replica set name is explicitly specified
-	opts.Direct = (opts.ReplicaSetName == "")
+	// Connect directly to a host if there's no replica set specified, or
+	// if the connection string already specified a direct connection.
+	opts.Direct = (opts.ReplicaSetName == "") || opts.Direct
```

Comment:
I think we'd decided to still treat the lack of a replica set name as a direct connection in tools, but I wanted to double-check that this is still the case.

Reply:
Yeah I think for now we shouldn't change that behavior. This looks good.